dbt_project.yml

```diff
@@ -32,7 +32,9 @@ clean-targets:         # directories to be removed by `dbt clean`
 # as tables. These settings can be overridden in the individual model files
 # using the `{{ config(...) }}` macro.
 models:
-  my_new_project:
-      # Applies to all files under models/example/
-      example:
-          materialized: view
+  jaffle_shop:
+    marts:
+        core:
+            +materialized: table
+    staging:
```

Comment:
Just a FYI, dbt actually defaults to materializing models as views so there is no need to explicitly config any models as views.